### PR TITLE
[TEL-300] Upgrade webclient to jitsi 2.0.5076

### PR DESCRIPTION
**Pull request by ivanjiang5628**
## Description
- [Jira Ticket](https://janeapp.atlassian.net/browse/TEL-300)

As part of our plan to improve the Telehealth video quality, we want to upgrade our Jitsi web app to version 5076.

- Checkout this branch from master and rebase the branch onto jitsi upstream [stable-5076 ](https://github.com/jitsi/jitsi-meet/commits/stable/jitsi-meet_5076) branch.

- Remove the "invite people button" UI that we don't need.

- Fix lint errors & re-format the interface_config.js file .
 
- Fix an issue that when the chat window is open, the timer appears twice. 
  - Cause: Jitsi previously placed the `Subject` component (to display conference time ) in the `Conference` component, with the 5076 build they have moved the `Subject` to the `LargeVideo` component. So we have 2 `Subject` comps on the main call page after rebasing.
  - Solution: Remove the `Subject` comp from the conference.js file after rebase on jitsi 5076 branch. [commit](https://github.com/janeapp/video-chat/pull/35/commits/9b51d0a081c7f39e32962e3ba05c0dedcc6e11f5#diff-9a5bbe4eeb74a0be25f1fd61f2bf943f2a7b2102c095d9c084815b1355794928L200)
   - Before
   

   - After
    

### General PR Class
👁 = UX / UI improvement
🐛 = Bug Fix (Fixes an Issue)

### Dependencies / ENV

- [ ] requires env configuration to be added in production
- [x] js package changes<sup>1</sup>
- [x] more than 200 LOC changed in production files<sup>1</sup>
- [ ] includes a user-facing workflow change to an existing production feature (user muscle memory or pattern recognition will be affected)
- [ ] could prevent access to Jane Video (eg. cors, middleware, changes to auth system)
- [ ] affects a widely used component or piece of code
- [ ] I have a doubt - I want the RMT to review this. If possible, please elaborate your concerns in the risk assessment section.

<sup>1</sup> No need to explain these risk factors below

### Release Risk Assessment
Should be Medium? 

## Code Review
Resource: [Dev Team Notion Page](https://www.notion.so/janeapp/Dev-Team-f06c6eb2ccca4066bc63fc1ac1bd2549)
Resource: [Code Review Checklist](https://www.notion.so/janeapp/Code-Review-checklist-2c510c527ac7470c902a5e8f25f9db3c)

- [x] I clearly explained the WHY behind the work, in the Description above

#### Design
- [x] I added instructions for how to test, in the QA section below
- [ ] I added specs for changes, or determined that none were required
- [x] I demoed this to the appropriate person
- [x] I considered both mobile & desktop views, or that wasn't relevant

#### Code
- [x] I committed code with informative git messages
- [x] I wrote readable code, or added comments if it was complex
- [x] I performed a self-review of my own code
- [ ] I rebased my branch on the latest `master`

## QA and Smoke Testing
Smoke test for the video chat app.

### Steps to Reproduce
Create an online appointment on S14 sandbox -> open the video chat -> please ensure everything works properly and the upgrade would not affect the current functionality or cause new issues.

### CODE REVIEW

**Review by calvinmcc — APPROVED** [review]
Obviously this is a tough one to review since so much got changed in Jitsi upgrades. But based on what I could see from the last two commits, I think we should be good to go. Since we can't verify everything code-wise (unless someone wants to go over all 1000 files changed), we should definitely QA this thoroughly.

File: bugsnag.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable */
+import bugsnag from '@bugsnag/js';
+
+const getBugsnagClient = () => {
+    let bugsnagClient;
+
+    if (process.env.NODE_ENV === 'production') {
+        bugsnagClient = bugsnag({
+            apiKey: 'a0c8e2c65bed338af650acd9c2192855',
```
Comment (gerbus) [inline bugsnag.js]: This probably should not be committed. Is there a way to keep this somewhere more secure?

File: react/features/toolbox/components/web/Toolbox.js [hunk 2]
```diff
@@ -1362,13 +1362,13 @@ class Toolbox extends Component<Props, State> {
...
-                            icon = { IconInviteMore }
-                            onClick = { this._onToolbarOpenInvite }
-                            tooltip = { t('toolbar.invite') } /> }
+                    {/*{ buttonsRight.indexOf('invite') !== -1*/}
+                    {/*    && <ToolbarButton*/}
+                    {/*        accessibilityLabel =*/}
+                    {/*            { t('toolbar.accessibilityLabel.invite') }*/}
+                    {/*        icon = { IconInviteMore }*/}
+                    {/*        onClick = { this._onToolbarOpenInvite }*/}
+                    {/*        tooltip = { t('toolbar.invite') } /> }*/}
```
Comment (gerbus) [inline react/features/toolbox/components/web/Toolbox.js:1371]: Should we just remove this, or did you want to keep it around?

**Review by gerbus — CHANGES_REQUESTED** [review]
Took a look at your commits, had a couple of questions

File: bugsnag.js (hunk 1 above)
Comment (ivanjiang5628) [reply bugsnag.js]: Yes, I think I can remove the key from the PR. I can store the key locally or copy it from the bugsnag project page.

Comment (gerbus) [reply bugsnag.js]: Yeah, I'm not too familiar with how this repo handles environment variables in production 🤷🏻

File: react/features/toolbox/components/web/Toolbox.js (hunk 2 above)
Comment (ivanjiang5628) [reply react/features/toolbox/components/web/Toolbox.js:1371]: Maybe we can keep it for now? I will create a new PR to clean up the unused components / re-format the code later

**Comment by jngo-janeapp** [comment]
Hey we've tested this multiples times before Christmas so this PR is 🚀 from the QA stand point.

File: react/features/toolbox/components/web/Toolbox.js (hunk 2 above)
Comment (gerbus) [reply react/features/toolbox/components/web/Toolbox.js]: Sounds good

File: bugsnag.js (hunk 1 above)
Comment (gerbus) [reply bugsnag.js]: @ivanjiang5628 is this just for tracking bugs in development? How will the key get inserted in production?

Comment (ivanjiang5628) [reply bugsnag.js]: It is for production we will not send bugsnag events in dev env. We don't have CD for this app so I need to insert the key manually before I build the prod tar ball file.

Comment (gerbus) [reply bugsnag.js]: I see. If this isn't in 1password yet, it probably should be. And I would think the `README.md` should outline how we build the prod tarball. That can be a follow-up PR I think.

**Review by gerbus — APPROVED** [review]

**Pull request merged**